docker_images/generic/tests/test_api.py

```diff
@@ -11,6 +11,7 @@
     "feature-extraction": ["osanseviero/fasttext_english"],
     "image-classification": ["osanseviero/fastai_cat_vs_dog"],
     "tabular-classification": ["osanseviero/wine-quality"],
+    # TODO "tabular-regression": [],
```

Comment:
why are you touching this under `generic`?

Reply:
Reverted.